source/components/molecules/NavigationButtonField/NavigationButtonField.tsx

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components/native';
-import { Button, Icon, Text } from '../../atoms';
-import Box from '../../atoms/Box/Box';
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components/native";
+import { Button, Icon, Text } from "../../atoms";
+import Box from "../../atoms/Box/Box";
 
 const ButtonFieldWrapper = styled(Box).attrs({
-  m: '2px',
+  m: "2px",
 })`
-  flex: 1;
+  flex: 0;
```

Comment:
Do we really need the `flex` property at all?

Reply:
Turns out we don't!